subsys/mgmt/ec_host_cmd/backends/ec_host_cmd_backend_spi_stm32.c

```diff
@@ -260,6 +266,24 @@ static int expected_size(const struct ec_host_cmd_request_header *header)
 	return sizeof(*header) + header->data_len;
 }
 
+#ifdef CONFIG_PM
+static void ec_host_cmd_pm_policy_state_lock_get(struct ec_host_cmd_spi_ctx *hc_spi)
+{
+	if (!hc_spi->pm_policy_lock_on) {
+		hc_spi->pm_policy_lock_on = true;
+		pm_policy_state_lock_get(PM_STATE_SUSPEND_TO_IDLE, PM_ALL_SUBSTATES);
+	}
+}
+
+static void ec_host_cmd_pm_policy_state_lock_put(struct ec_host_cmd_spi_ctx *hc_spi)
+{
+	if (hc_spi->pm_policy_lock_on) {
+		hc_spi->pm_policy_lock_on = false;
+		pm_policy_state_lock_put(PM_STATE_SUSPEND_TO_IDLE, PM_ALL_SUBSTATES);
+	}
+}
+#endif /* CONFIG_PM */
```

Comment:
How about definining the functions to a no-op in the else case? Saves you the guards on cthe callers

Reply:
Thanks, done.